edx-platform/nau-basic/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -3,4 +3,6 @@
 @import 'footer';
 @import 'responsive';
 @import 'forms';
-@import 'courses';
```

Comment:
mmmm I'm not sure what is this :(
![2021-06-03_07-01](https://user-images.githubusercontent.com/64440265/120654077-55731d00-c44f-11eb-973b-01432f841c38.png)

Reply:
Ya estaba en master no entiendo porque me  lo marca como algo nuevo
 https://github.com/fccn/nau-themes/blob/master/edx-platform/nau-basic/lms/static/sass/partials/lms/theme/_extras.scss

Reply:
You added a blank line at the end (L#8)

Reply:
Thanks Moisés, What is your advice? It is not clear to me that I should change here

Reply:
Just delete the blank line. That sign means that there is no `\n` character after `@import 'courses';`. Some editors add that automatically. It doesn't really make any difference.